recipes/pffft/all/test_package/test_package.c

```diff
@@ -0,0 +1,7 @@
+#include <pffft.h>
+
+int main() {
+
+    int s = pffft_simd_size();
```

Comment:
this may produce a warning for unused variable, I usually add code to print result (`printf`)
not needed to fix, just FYI